tests/unit/server/api/experiments/attributes/[id].get.test.ts

```diff
@@ -0,0 +1,81 @@
+import { describe, expect, vi, it } from "vitest"
+import { v4 as uuidv4 } from "uuid"
+import type { H3Event, EventHandlerRequest } from "h3"
+import getAttribute from "~~/server/api/experiments/attributes/[id].get"
+
+describe("Api Route GET /api/experiments/attributes/{id}", () => {
+  it("should get an Attribute by id", async () => {
+    const mockAttribute = {
+      id: uuidv4(),
+      name: "Test Attribute",
+      values: [
+        { id: uuidv4(), value: "Value 1", toList: () => "Value 1" },
+        { id: uuidv4(), value: "Value 2", toList: () => "Value 2" },
+      ],
+      toDetail: (valueList: string[]) => ({
+        id: mockAttribute.id,
+        name: mockAttribute.name,
+        valueList,
+      }),
+    }
+
+    // Mocking Prisma
+    prisma.experimentAttribute.findFirst = vi.fn().mockImplementation(({ where }) => {
+      if (where.id === mockAttribute.id) {
+        return Promise.resolve(mockAttribute)
+      }
+      return Promise.resolve(null)
+    })
+
+    const event = {
+      context: {
+        params: {
+          id: mockAttribute.id,
+        },
+      },
+    } as unknown as H3Event<EventHandlerRequest>
+
+    const response = await getAttribute(event)
+
+    expect(response).toStrictEqual({
+      id: mockAttribute.id,
+      name: mockAttribute.name,
+      valueList: ["Value 1", "Value 2"],
```

Comment:
The value list should then be a list of objects with id and name like in the post test